website/docs/r/group.html.markdown

```diff
@@ -53,6 +52,10 @@ resource "opennebula_group" "group" {
             running_vms = 1
         }
     }
+
+    lifecycle {
+        prevent_destroy = true
+    }
```

Comment:
is it necessary here ?
Maybe that only a small comment besides the `delete_one_destruction` attribute would be ok ?